otx/algorithms/classification/adapters/mmcls/models/classifiers/semisl_classifier.py

```diff
@@ -32,7 +32,7 @@ def forward_train(self, imgs, **kwargs):
             raise ValueError("'gt_label' does not exist in the labeled image")
         if "extra_0" not in kwargs:
             raise ValueError("'extra_0' does not exist in the dataset")
-        target = kwargs["gt_label"].squeeze()
+        target = kwargs["gt_label"].squeeze(dim=1)
```

Comment:
How about checking the tensor shape? before this line?

```python3
if kwargs["gt_label"].shape == ??:
    target = kwargs["gt_label"].squeeze()
elif ?? :
    target = kwargs["gt_label"].squeeze(dim=1)
else:
```

Reply:
It could be. But does the case where gt label isn't one dimension exist?

Reply:
I'm not sure, hmm, then I just wonder about the previous implementation. 
Why the previous implementation used ```.squeeze()``` instead of ```.squeeze(dim=1)```.
However, it's not crucial, let's keep it as-is.